python/google/protobuf/internal/descriptor_test.py

```diff
@@ -121,6 +143,13 @@ def testContainingTypeFixups(self):
   def testContainingServiceFixups(self):
     self.assertEqual(self.my_service, self.my_method.containing_service)
 
+  @unittest.skipIf(
+    api_implementation.Type() != 'cpp',
+    'GetDebugString is only available with the cpp implementation',
+  )
+  def testGetDebugString(self):
+    self.assertEqual(self.my_file.GetDebugString(), TEST_FILE_DESCRIPTOR_DEBUG)
+
```

Comment:
shouldn't this test be executed in python too as this PR merged that feature?
(sry just needed that feature and debugged on an older python-protobuf and stumbled over this PR, pls just ignore if everything is as it should be :)!